api/request_source/request_source_plugin.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package nighthawk.request_source;
+
+import "google/protobuf/wrappers.proto";
+import "validate/validate.proto";
+import "api/client/options.proto";
+
+// Configuration for DummyPluginRequestSource (plugin name: "nighthawk.dummy-request-source-plugin")
+// that does nothing
+message DummyPluginRequestSourceConfig {
+  // Dummy input value.
+  google.protobuf.DoubleValue dummy_value = 1;
+}
+
+// Configuration for RPCPluginRequestSource (plugin name: "nighthawk.rpc-request-source-plugin")
+// that uses rpc
+message RPCPluginRequestSourceConfig {
+  string uri = 1;
```

Comment:
Can we add a comment explaining this field? All fields in proto messages should have good comments since they essentially represent the public API. Please apply throughout.